src/axom/quest/readers/C2CReader.cpp

```diff
@@ -965,56 +457,39 @@ void C2CReader::getLinearMeshNonUniform(
   // Clamp the lower error bound so it does not get impractically small.
   percentError = axom::utilities::clampLower(percentError, 1.e-10);
 
-#ifdef AXOM_DEBUG_WRITE_ERROR_CURVES
-  // Make some curves for debugging.
-  FILE* ferr = fopen("error.curve", "wt");
-
-  FILE* fhcl = fopen("hicurvelen.curve", "wt");
-  fprintf(fhcl, "# hicurvelen\n");
-
-  FILE* fcl = fopen("curvelen.curve", "wt");
-  fprintf(fcl, "# curvelen\n");
-
-  FILE* fthresh = fopen("threshold.curve", "wt");
-  fprintf(fthresh, "# threshold\n");
-#endif
```

Comment:
@BradWhitlock -- as far as I could tell, these were only creating files with a single line, so I removed it and the associated compiler define.

Perhaps they were used more in previous debugging/development code and the functionality was removed?

Reply:
I don't recall the current state of those but it may be that you'd only see anything interesting if dynamic sampling is turned on via the shaping driver.

Reply:
I can add it back if you'd like.

A quick search seems to indicate that it's restricted to this file: 
https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/llnl/axom+AXOM_DEBUG_WRITE_ERROR_CURVES&patternType=keyword&sm=0

Reply:
No, it looks like whatever else it was writing has been removed already. Deleting code is good, especially if it is not complete anymore.